services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/IncidentTest.java

```diff
@@ -0,0 +1,71 @@
+package com.mapbox.api.directions.v5.models;
+
+import com.mapbox.core.TestUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class IncidentTest extends TestUtils {
+
+  @Test
+  public void sanity() {
+    assertNotNull(buildDefaultIncident());
+  }
+
+  @Test
+  public void testSerializable() throws Exception {
+    Incident incident = buildDefaultIncident();
+    byte[] serialized = TestUtils.serialize(incident);
+    assertEquals(incident, deserialize(serialized, Incident.class));
+  }
+
+  @Test
+  public void testFromJson() {
+    String incidentJsonString = "{"
+        + "\"incident_type\": \"road_closure\","
+        + "\"start_time\": 1598575905,"
```

Comment:
Be aware, this was just updated to be a string in ISO8601 format. Same for the other time properties.

Reply:
@PureW could you please provide a used format?

Reply:
@LukasPaczos should we parse incoming String to a Date? or just store it as a String?

Reply:
```
{
          "id": 2448,
          "creation_time": "2020-08-12T14:16:40Z",
          "start_time": "2020-08-12T14:18:20Z",
          "end_time": "2020-08-12T14:46:40Z",
          "incident_type": "accident"
}
```
from (still unmerged) https://github.com/mapbox/valhalla-internal/pull/1015/files#diff-2efe098258521381fd0b77a16f4d8b27R1723

Reply:
fixed